srcpkgs/novaboot/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'novaboot'
+pkgname=novaboot
+version=20180323
+revision=1
+noarch=yes
+hostmakedepends="perl"
+makedepends="perl"
+depends="perl perl-Expect"
+short_desc="Tool that automates booting of operating systems on hardware or in qemu"
+maintainer="adam <ametisf@gmail.com>"
+license="GPL-2.0-only"
+homepage="https://github.com/wentasah/novaboot"
+distfiles="https://github.com/wentasah/novaboot/archive/${version}.tar.gz"
+checksum=f1a8781d9bb470d631014063986fb89334747bb718e3acef33d24a35f4359548
+
+do_build() {
+	make
```

Comment:
this can be replaced with the gnu-makefile build_style

Reply:
Fixed. Thank you.